app/controllers/movies_controller.rb

```diff
@@ -1,4 +1,6 @@
 class MoviesController < ApplicationController
+  # protect_from_forgery with: :exception
+
```

Comment:
I also wasn't sure if it should be included here as well, or just here. See note on file below.